ipi/engine/thermostats.py

```diff
@@ -544,7 +561,12 @@ def bind(self, beads=None, atoms=None, pm=None, nm=None, prng=None, fixdof=None)
 
         t = self._thermos[0]
         t.bind(pm=(nm.pnm[0, :], nm.dynm3[0, :]), prng=self.prng, fixdof=fixdof)
-        dpipe(dself.temp, dd(t).temp)
+        if self.pilect > 0.0:  # trying to pipe different temperature here
+            dpipe(dself.pilect, dd(t).temp)
+        else:
+            dpipe(dself.temp, dd(t).temp)
+        #        dpipe(dself.temp, dd(t).temp)
```

Comment:
If this work, pls clean up commented out lines like this, so we avoid accumulating noise in the codebase

Reply:
Done